wgsl/index.bs

```diff
@@ -748,7 +748,7 @@ type_decl
   | UINT32
   | VEC2 LESS_THAN type_decl GREATER_THAN
   | VEC3 LESS_THAN type_decl GREATER_THAN
-  | VEC3 LESS_THAN type_decl GREATER_THAN
+  | VEC4 LESS_THAN type_decl GREATER_THAN
```

Comment:
that change looks familiar :)

Reply:
Did it get done somewhere else and I missed it?

Reply:
No, I think I noticed it a while ago when writing the WGSL parser and forgot to make a fixing PR here